README.md

```diff
@@ -249,18 +266,36 @@ prevent concurrent, unsafe writes to the same file. Our current approach is
 therefore quite safe yet restrictive. We may revisit this decision based on
 developer feedback.
 
+### Quota
+
+Storage Foundation achieves fast and predictable performance by implementing its
+own quota management system.
+
+The web application must explicitly ask for capacity before storing any new
+data. This request will be granted according to the browser's quota
+guidelines. The application can then use the capacity to execute write
+operations. The application can also delete or overwrite data to make space for
+new data without issuing a new request.
+
+This means that anytime an application starts a new Javascript execution context
+(e.g., new tab, new worker, reloading the page), it must make sure it owns
+sufficient capacity before writing new data. It must therefore either
+delete/shrink old files or request additional capacity.
```

Comment:
Consider explicitly mentioning overwritring old files as well e.g. not delete or shrink an existing file, but always write within the existing boundaries.

Reply:
Done